packages/cli/readme.md

```diff
@@ -25,21 +25,32 @@ The following commands are provided by hops-cli:
 - `hops develop` - starts a Webpack development server
 - `hops serve` - initiates a project build, stores the build artifacts in the file system and starts a production (Express) server
 - `hops start` - if NODE_ENV is set to production, this runs `hops serve`. Otherwise, `hops develop` gets executed
-- `hops start --production` - this overrides the NODE_ENV setting and runs `hops serve`
+
+Please note that hops-cli is not meant to be called directly, but rather to be added to your project's package.json file and then called indirectly by using npm or yarn (see below).
+
+## Static mode
+hops-cli can be used as a static site generator, too. To enable static mode, pass `--static` or `-s` to the above commands and configure a `locations` array to your package.json file.
```

Comment:
`to the above commands` -> That sounds like `--static` also used for `develop`, which isn't the case afair.